packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -395,6 +417,23 @@ const App = (props) => {
                         </Box>
                     </CurrencyProvider>
                 </IntlProvider>
+                {ACTIVE_DATA_ENABLED && (
+                    <script
+                        type="text/javascript"
+                        src={getAssetUrl('static/dwanalytics-22.2.js')}
+                        id="dwanalytics"
+                        async="async"
+                        onLoad={trackPage}
```

Comment:
```suggestion
```
I've tried a few different things, and I haven't been able to get this `onLoad` to trigger. But that shouldn't matter, the `trackPage` effect seems to cover the same "initial load" use case:
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/7d6dead4dc741aa4dbd413a3c7827925eb5ed8a6/packages/template-retail-react-app/app/components/_app/index.jsx#L280-L282